consultation_analyser/support_console/views/users.py

```diff
@@ -31,15 +33,32 @@ def new(request: HttpRequest):
 def show(request: HttpRequest, user_id: int):
     user = get_object_or_404(User, pk=user_id)
     consultations = Consultation.objects.filter(users__in=[user])
+    dashboard_group = Group.objects.get(name=DASHBOARD_ACCESS)
 
     if not request.POST:
-        form = EditUserForm({"user_id": user_id, "is_staff": user.is_staff})
+        form = EditUserForm(
+            {
+                "user_id": user_id,
+                "is_staff": user.is_staff,
+                "dashboard_access": user.has_dashboard_access,
+            }
+        )
     else:
         form = EditUserForm(request.POST, current_user=request.user)
         if form.is_valid():
             is_staff = form.cleaned_data["is_staff"]
             user.is_staff = is_staff
             user.save()
+
+            assign_dashboard_access = form.cleaned_data["dashboard_access"]
+            if assign_dashboard_access:
+                user.groups.add(dashboard_group)
```

Comment:
I take it this doesn't have any unexpected behaviour if the user already has access to this group?

Reply:
Good question - I should test (I don't think so).

Reply:
If we try and add the same group more than once then it has no effect - but good to check.